src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp

```diff
@@ -136,7 +140,16 @@ std::string getEntityAttributeValue
       }
       else
       {
-        TIMED_RENDER(answer = attribute.pcontextAttribute->getValue());
+
+        if (attributeType == DATE_TYPE)
+        {
+          answer = isodate2str(attribute.pcontextAttribute->numberValue);
+        }
+        else
+        {
+          answer =  attribute.pcontextAttribute->getValue();
+        }
+        TIMED_RENDER(answer = answer);
```

Comment:
We just got really fast here ... :-)
I imagine the TIMED_RENDER macro should be used for the two assignments of answer.